test/acceptance/test/add_tests.go

```diff
@@ -49,19 +55,19 @@ var _ = Describe("Weave GitOps Add App Tests", func() {
 		var exitCode int
 		private := true
 		tip := generateTestInputs()
-		appRepoRemoteURL := "ssh://git@github.com/" + githubOrg + "/" + tip.appRepoName + ".git"
+		appRepoRemoteURL := "ssh://git@" + gitURLName + ".com/" + gitOrg + "/" + tip.appRepoName + ".git"
```

Comment:
Please rename this var to `gitProviderName` or something similar just to align more with the value it contains. It would help for readability.